src/tests/TestCryptographicRandom/TestCryptographicRandom.pro

```diff
@@ -12,6 +12,14 @@ CONFIG += testcase
 CONFIG += qt warn_on
 CONFIG -= app_bundle
 LANGUAGE = C++
+
+# We build this test with 'gui' in QT,
```

Comment:
I’m not quite sure I understand what that means. We build other tests without gui, but this one with gui? Why do we do that? This doesn't seem to have any more GUI than the other tests.